nextcloudappstore/core/views.py

```diff
@@ -270,3 +273,42 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['form'] = AppRegisterForm()
         return context
+
+
+class AppEditView(LoginRequiredMixin, TemplateView):
```

Comment:
There's probably a better base class for an edit and detail view. Your current code needs to fetch the model from the context which is kinda meh and you also need to handle http methods and non existing objects with custom 404 code